articles/tutorials/data-scenarios/index.md

```diff
@@ -0,0 +1,257 @@
+
+### Performance
+
+Beyond scale, keeping your authentication data separate from any other data in your app is important because the two sets of data are also likely accessed with different frequencies. The Auth0 data store is not optimized to be queried with extremely high frequency. It is better to leave this up to a specialized database service, optimized for fast, large data operations.
+
+### Flexibility 
+
+By using a separate database, your access to your users' authentication data is concentrated in a small chunk of code, and your access to all other data is grouped separately. Your organizational demands may not be met by the Auth0 data store because it is built to accomodate only the user profiles and their Metadata. Certain actions that you might require from a customizable database service are not possible in Auth0 data store. For example, if you wanted to make a specific query like `SELECT users.favGenre, access.roles FROM users, access WHERE users.user_id = access.user_id`. This is something you can't do with in the auth0 data store. Using a separate database will allow you to manage your data however you see fit.
+
+### Example
+
+Here is an example of data that is associated with a user but not with authenticating that user in the app: In the case of our mobile music application, the user’s music needs to be saved, so they can find it easily when they log in again. This data is not required in the process of authenticating the user for the app, but favorite songs and artists are personal data that should be associated with the user. So we would want to store this data in a separate database connected to the backend of our mobile app, instead of in the Auth0 data store. Here is how we did this:
+
+Using the user’s unique `user_id` from their Auth0 [user profile](/user-profile/normalized#storing-user-data), we can make sure the data is associated with the user in our separate database in order to query by a unique identifier. Here is an example row from the `songs` table in our Heroku Postgres database:
+
+| song_id   | songname           | user_id                    |
+| --------- | ------------------ | -------------------------- |
+| "1a"      | "Yellow Submarine" | "google-oauth2|xxxyyy123" |
+
+The Node.js backend authenticates requests to the specific URI associated with getting the user’s personal data from the database. This is accomplished through the validation of a JSON Web Token. [Learn about token based authentication and how to easily implement JWT in your applications.](https://auth0.com/learn/token-based-authentication-made-easy/) 
```

Comment:
@eharkins This reference is great, but I would add it as a note:

[Learn about token based authentication and how to easily implement JWT in your applications.](https://auth0.com/learn/token-based-authentication-made-easy/)